Example/Bellerophon/AppDelegate.swift

```diff
@@ -15,12 +15,23 @@ import Bellerophon
 class AppDelegate: UIResponder, UIApplicationDelegate, BellerophonManagerProtocol {
 
     var window: UIWindow?
-
+    var killSwitchURL: String?
```

Comment:
I need to `assert` on `killSwitchURL` in the delegate method, so added it as an instance var.